package.json

```diff
@@ -25,13 +25,11 @@
     "exports": "eyeglass-exports.js"
   },
   "scripts": {
-    "test": "npm run --silent test-libsass && npm run --silent test-ruby",
+    "test": "npm run test-libsass && npm run test-ruby",
```

Comment:
Any reason you removed the `--silent` flag? npm erroring like crazy does not really help most of the time.